Fixes (#1)
mOctave/searchers: data/searchers/map planets.txt

planet Aes
	attributes searchers mining
	landscape land/badlands4
	description `Aes is a very strange world, completely devoid of life. Although there is an atmosphere, it is completely unbreathable, and the dry heat is oppressive. This planet should not be habitable, yet the Searchers have managed to live here, building giant domes on the surface and sealed caves underneath. Perhaps the reason this planet has been settled is that it is full of useful ores: the entire planet amounts to one big mining operation for iron, nickel, and gold.`

> 	spaceport `The spaceport here looks to your eyes a lot like pictures you've seen of bases on Mars prior to its terraforming: undeniably mechanical, designed to withstand hostile climates, and very, very old. The landing pads are basically giant airlocks: you land on a ring that closes the dome, and then the entire chamber is pumped full of breathable air. When a ship takes off, it first must go through the several-minute long process of pumping the air back out again.`

Reviewer: ```suggestion
	spaceport `The spaceport here looks a lot like pictures you've seen of bases on Mars prior to its terraforming: undeniably mechanical, designed to withstand hostile climates, and very, very old. The landing pads are basically giant airlocks: you land on a ring that closes the dome, and then the entire chamber is pumped full of breathable air. When a ship takes off, it first must go through the several-minute long process of pumping the air back out again.`
```

	landscape land/clouds_02
	description `Salus is a very cloudy world, notable for its heat and the red tint that the cloud cover gives its sun. The largest producer of medical products of all Searcher planets, it is very rich, and boasts the only major military base in all of Searcher space. Your ship is scanned upon landing, and you are carefully watched to make sure you don't attempt to do anything illegal.`
	spaceport `The spaceport on Salus is very large, dominated by the military base. There is a lot of traffic here, but it has managed to avoid the worst of the congestion by the use of an extraordinarily complicated transportation network. The landing pads are generously sized, and there are enough if them to land even the very largest of fleets. As soon as you landed, you were provided with a guide instructing you in the operation of the transportation network.`

> 	security 1

Reviewer: I'm going to leave this out for now, but thank you for reminding me that I never did assign security or tribute data to any of these planets.

```suggestion
```

Author: It says that you can't do anything without being watched though.

Reviewer: Yeah, I've added a "security" 0.9 on it in my last commit. I _might_ bump it up to 1, since it fits the description better, but I might also just keep it as-is since the only current factions with such high security are the Hai, Quarg and Pug. The Searchers can do a lot of scans, but they can't necessarily recognize alien illegal stuff for what it is.

planet Tela
	attributes searchers textiles
	landscape land/mountain9

> 	description `Tela is the largest cloth producer of Searcher space, despite its size. It is covered in massive textile mills, and under its surface are hidden thousands upon thousands of young Searchers who labor daily producing cloth. Tela's population never sees most of the cloth it produces again: it is shipped elsewhere, to where it can sell for a good deal of money. Because of this, Tela is a very poor world, whose inhabitants have to constantly struggle to survive.`

Reviewer: The use of "labour" was actually intentional here. I do pretty much all of my work in Canadian English because it seems more natural to me.

But since this is aiming for vanilla, you're probably right that it needs to be switched.

```suggestion
	description `Tela is the largest cloth producer of Searcher space, despite its size. It is covered in massive textile mills, and under its surface are hidden thousands upon thousands of young Searchers who labor daily producing cloth. Tela's population never sees most of the cloth it produces again: it is shipped elsewhere, to where it can sell for a much higher price. Because of this, Tela is a very poor world, whose inhabitants have to constantly struggle to survive.`
```
